### DISCUSSION TITLE
Semantic Versioning

### DISCUSSION THREAD

**Discussion by Corepex** [7nohe/openapi-react-query-codegen#87] [General]
Since this repo uses three-digit version numbers, I assume that it has something to do with semantic versioning.
If this repo decides to use semantic versioning, we should focus more on milestones and potential BC-breaks 

WDYT?

**Comment by seriouslag** [comment]
Yes, I believe we released version v1 a little soon, we switched to a new service generator for version 1.0.0. We fixed a lot of bugs doing so, and now have support for that library, but the service generator is not yet in version one. Its API is changing almost daily. We have been getting a lot of issue because of the switch, either finding bugs with the underlying generator or our integration with it, instead of releasing many major releases we have been fixing those issues as patches and features. We now have a near complete unit and integration tests suite. (We could use a few more tests) (especially for the CLI calls) which should help us prevent breaking changes in the future.
